### ISSUE TITLE
Documentation unclear surround GitHub Desktop installation and its inclusion (or not) of Git

### ISSUE THREAD

**Issue by spikysimonw** [github/training-kit#920]
## Overview
**TL;DR**
Confusing and misleading information on a) whether or not Git is included as part of a GitHub Desktop install, and b) whether Git is already installed on "most versions of MacOS".

### Extra Detail

- Information in para [A](https://github.com/github/training-kit/blob/master/git-guides/install-git.md#install-git-using-github-desktop) conflicts with instruction within para [B](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/installing-and-authenticating-to-github-desktop/setting-up-github-desktop#part-3-configuring-git): _"Installing GitHub Desktop will also install the latest version of Git if you don't already have it."_ vs _"You must have Git installed before using GitHub Desktop."_

- Para [C](https://github.com/github/training-kit/blob/master/git-guides/install-git.md#install-git-on-mac) states _"Most versions of MacOS will already have Git installed"_ whereas this is only true if Apple's Command Line Developer Tools are optionally installed (and which I suspect is less likely to be the case for a user looking at GitHub Desktop installation instructions).

#### Screenshots
Link A - https://github.com/github/training-kit/blob/master/git-guides/install-git.md#install-git-using-github-desktop
Link B - https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/installing-and-authenticating-to-github-desktop/setting-up-github-desktop#part-3-configuring-git
Link C - https://github.com/github/training-kit/blob/master/git-guides/install-git.md#install-git-on-mac
 
#### Reasoning
Testing suggests a local git install does **not** get installed as part of an official GitHub Desktop installation on macOS, however instead it seems some form of sandboxed implementation is included (?) as within the GitHub Desktop app itself it can perform git operations without any local git install being available. Suggest this needs to be clarified and documented, ie. if and/or when a separate local installation of git is required.

**Comment by MagnusOxlund** [comment]
GitHub Desktop for macOS does in fact come with git. It's located inside the application package at `GitHub Desktop.app/Contents/Resources/app/git/bin/git`. It doesn't seem to be configured for direct CLI use, however. For instance, issuing a `git clone` will complain about a missing templates directory:

`warning: templates not found in //share/git-core/templates`

**Comment by ktbyers** [comment]
Yeah, I thought this was very confusing as well.
